src/index.html

```diff
@@ -4,9 +4,347 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="cards">
+        <div class="card-block cards__card" data-qa="card">
+          <img
+            class="card-block__image"
+            src="./images/imac.jpeg"
+            alt="Apple iMac 27-inch"
+          >
+          <h1 class="card-block__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="card-block__code">Код товара: 195434</p>
+
+          <div class="card-block__stars">
+            <div class="stars stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <section class="card-block__feedbacks">Отзывов: 5</section>
+          </div>
+
+          <section class="card-block__price">
+            <div class="card-block__price-price">Цена:</div>
+            <div class="card-block__price-sum">69 999 грн</div>
+          </section>
+
+          <a
+            class="card-block__buy"
+            data-qa="card-hover"
+            href="#buyiMac27"
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="card-block cards__card">
+          <img
+            class="card-block__image"
+            src="./images/imac.jpeg"
+            alt="Apple iMac 27-inch"
+          >
+          <h1 class="card-block__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="card-block__code">Код товара: 195434</p>
+
+          <div class="card-block__stars">
+            <div class="stars stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <section class="card-block__feedbacks">Отзывов: 5</section>
+          </div>
+
+          <section class="card-block__price">
+            <div class="card-block__price-price">Цена:</div>
+            <div class="card-block__price-sum">69 999 грн</div>
+          </section>
+
+          <a
+            class="card-block__buy"
+            href="#buyiMac27"
+          >
+            Купить
+          </a>
+        </div>
+
+        <div class="card-block cards__card">
+          <img
+            class="card-block__image"
+            src="./images/imac.jpeg"
+            alt="Apple iMac 27-inch"
+          >
+          <h1 class="card-block__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
```

Comment:
h1 is main heading of the page
use h2-h6 for this part

Reply:
Agree, change this